edenia/antelope-tools
# Chore/adjust margins and refactor code

### Correct margin spaces

### What does this PR do?

- Resolve #1117. 
- Refactor code.
- Center the footer logos vertically.
- Use the social media images with the right dimensions.

### Steps to test

1. Open docs/index.html.
1. Check the hompage in mobile breakpoints.

### Screenshots

iPhone 12 (breakpoint under 768 px)
![imagen](https://user-images.githubusercontent.com/66583677/212139344-960990ee-fb33-46f6-8339-4547106b85a7.png)

![imagen](https://user-images.githubusercontent.com/66583677/212138695-2a62ead4-2a1f-4ae1-9103-8f5c586461dd.png)

iPad Mini (breakpoint between 768 px and 1280 px)
![imagen](https://user-images.githubusercontent.com/66583677/212138840-9c50be46-b76a-4696-9fcf-a19255a10509.png)
